tests/entities_column_value_simple_tests.py

```diff
@@ -251,3 +251,186 @@ def test_should_set_number_column_value_to__a_valid_string_value():
+
+    #Assert
+    eq_(column_value.value,'Done')
+    eq_(format['index'], 1)
+
+@raises(e.ColumnValueError)
+def test_should_set_invalid_integer_to_status_index_value():
+    # Arrange
+    id = 'status_1'
+    title = "status"
+    column_type = ColumnType.status
+    index_value = {
+        'index': 123,
+    }
+    settings = {'labels': {'0': 'Working on it', '1': 'Done', '2': 'Stuck'}}
+    settings_str = json.dumps(settings)
+    value = json.dumps(index_value)
+    column_value = en.cv.create_column_value(column_type, id=id, title=title,settings_str=settings_str)
+
+    # Act 
+
+    column_value.value = value
+
+def test_should_set_a_status_column_value_with_valid_integer_index_value():
+
+    # Arrange
+    id = 'status_1'
+    title = "status"
+    column_type = ColumnType.status
+    index_value = {
+        'index': 1,
+    }
+    settings = {'labels': {'0': 'Working on it', '1': 'Done', '2': 'Stuck'}}
+    settings_str = json.dumps(settings)
+    value = json.dumps(index_value)
+    column_value = en.cv.create_column_value(column_type, id=id, title=title,value=value,settings_str=settings_str)
+
+    # Act
+    format = column_value.format()
+
+    #Assert
+    eq_(column_value.value,'Done')
+    eq_(format['index'], 1)
+
+@raises(e.ColumnValueError)
+def test_should_set_invalid_string_to_status_index_value():
+    # Arrange
+    id = 'status_1'
+    title = "status"
+    column_type = ColumnType.status
+    index_value = {
+        'index': "123",
+    }
+    settings = {'labels': {'0': 'Working on it', '1': 'Done', '2': 'Stuck'}}
+    settings_str = json.dumps(settings)
+    value = json.dumps(index_value)
+    column_value = en.cv.create_column_value(column_type, id=id, title=title,settings_str=settings_str)
+
+    # Act 
+
+    column_value.value = value
+
+def test_should_set_a_status_column_value_with_valid_integer_index_value():
+
+    # Arrange
+    id = 'status_1'
+    title = "status"
+    column_type = ColumnType.status
+    index_value = {
+        'index': 1,
+    }
+    settings = {'labels': {'0': 'Working on it', '1': 'Done', '2': 'Stuck'}}
+    settings_str = json.dumps(settings)
+    value = json.dumps(index_value)
+    column_value = en.cv.create_column_value(column_type, id=id, title=title,value=value,settings_str=settings_str)
+
+    # Act
+    format = column_value.format()
+
+    #Assert
+    eq_(column_value.value,'Done')
+    eq_(format['index'], 1)
+
+def test_should_set_a_status_column_value_with_valid_string_label_value():
+
+    # Arrange
+    id = 'status_1'
+    title = "status"
+    column_type = ColumnType.status
+    index_value = {
+        'index': 1,
+    }
+    settings = {'labels': {'0': 'Working on it', '1': 'Done', '2': 'Stuck'}}
+    settings_str = json.dumps(settings)
+    value = json.dumps(index_value)
+    column_value = en.cv.create_column_value(column_type, id=id, title=title,value=value,settings_str=settings_str)
+
+    # Act
+    column_value.value='Done'
```

Comment:
This is how you should be doing these tests.

I would only then recommend that you start out with an empty column_value (with no value) and only assert against column_value.value. No need to test _format_.  That will be tested with the two initial tests unless there is specific error logic to be tested.  That will always be called out in the documentation.